domains/pallets/auto-id/src/lib.rs

```diff
@@ -234,6 +234,14 @@ mod pallet {
     #[pallet::storage]
     pub(super) type AutoIds<T> = StorageMap<_, Identity, Identifier, AutoId, OptionQuery>;
 
+    /// Stores list of revoked certificates.
+    ///
+    /// It maps the unique identifier to the serial number of the certificate. Before accepting
+    /// the certificate, external entities should check if the certificate or its issuer has been revoked.
+    #[pallet::storage]
+    pub(super) type CertificateRevocationList<T> =
```

Comment:
This seems incorrect. An Issuer can issue multiple certs and they will have multiple Serials. So for each Issuer, you should Store the List of Serials instead